src/js_native_api_v8.h

```diff
@@ -55,6 +57,8 @@ struct napi_env__ {
   }
 
   v8impl::Persistent<v8::Value> last_exception;
+  v8impl::RefTracker::RefList reflist;
```

Comment:
I think a comment here about why there are 2 lists, along with how we need to delete those in the first list before the second would be good here.   I don't remember the full explanation and I think it would be good to capture it here.